src/main/java/org/jenkinsci/plugins/workinghours/utils/DateTimeUtility.java

```diff
@@ -70,8 +70,12 @@ public static LocalTime localTime(final String value) {
         return null;
     }
 
-    public static LocalTime localTimeFromMinutes(final int minutes) {
-        return LocalTime.of(minutes / 60, minutes % 60);
+    public static LocalTime localTimeFromMinutes(int minutes) {
+
+        if (minutes >= 1440) {
```

Comment:
Can you add a comment about this? I'm not sure why it's being done :-)

Reply:
Could also use some constant for 1440, e.g. `MINUTES_IN_24_HOURS`?

Reply:
Is this code expected to work if a large value such as 2880 gets passed in?

Reply:
Java LocalTime's hour couldn't be larger than 23, and in order to show the user in the webapp, that the end of the day is 24:00, that's 1440 minutes, and the 1440 will get's transferred to the plugin, so I had to low it down to 1439 to indicate the end of the day.

Reply:
Yeah, I should use a const